.github/workflows/integration.yml

```diff
@@ -0,0 +1,38 @@
+name: .NET Build & Integration Test
+
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  integration:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 5.0.x
```

Comment:
Do we want to update this to 6.0.x?

Reply:
This reflects the project's .NET version, which would be good to upgrade but should likely be a separate task. Want to create an issue for it?

Reply:
Sure thing!